pkg/controller/flow/reconcile.go

```diff
@@ -333,3 +324,33 @@ func (r *reconciler) getDefaultClusterBusName() (string, error) {
 
 	return fallbackClusterBusName, nil // return the fallback value
 }
+
+func (r *reconciler) getFeed(flow *v1alpha1.Flow) (*feedsv1alpha1.Feed, error) {
+	feedList := &feedsv1alpha1.FeedList{}
+	err := r.client.List(
+		context.TODO(),
+		&client.ListOptions{
+			Namespace:     flow.Namespace,
+			LabelSelector: labels.Everything(),
+			// TODO this is here because the fake client needs it. Remove this when it's no longer
+			// needed.
+			Raw: &metav1.ListOptions{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: feedsv1alpha1.SchemeGroupVersion.String(),
+					Kind:       "Feed",
+				},
+			},
+		},
+		feedList)
+	if err != nil {
+		glog.Errorf("Unable to list feeds: %v", err)
+		return nil, err
+	}
+	for _, feed := range feedList.Items {
+		ownerRef := metav1.GetControllerOf(&feed)
+		if ownerRef != nil && ownerRef.Name == flow.Name && ownerRef.UID == flow.UID {
```

Comment:
Do you need to compare both `flow.Name` and `flow.UID`?

Do you want/need to ensure that `flow.UID` is not default-valued?